### PR TITLE
Updates the OIN Manager guide to include deleting submissions

### PR DESCRIPTION
**Pull request by IanHakes-okta**
## Description:
- **What's changed?** 
Instructions for deleting draft submissions and published submissions.

- **Is this PR related to a Monolith release?** 
Yes. 2020.08.1

### Resolves:

* [OKTA-315485](https://oktainc.atlassian.net/browse/OKTA-315485)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md [hunk 1]
```diff
@@ -19,3 +19,31 @@ Similarly, when you enable a new capability in your application (for example, ad
...
+## Delete draft submissions
+
+There are two scenarios where you need to delete a draft submission:
+
+1. You have 10 draft submissions, which is the maximum number permitted in the OIN Manager.
+1. You have decided against completing a draft submission and want to remove it.
+
+In either of these scenarios, the OIN Manager provides a method to delete unpublished submissions. For instructions on how to delete app integrations that are already published in the OIN catalog, see [Delete published submissions](#delete-published-submissions).
+
+You can only delete unpublished submissions that are in **DRAFT** state. If the delete icon is dimmed, that means the delete operation isn't available for that submission. If you can't move your submission to the **DRAFT** state, contact the Okta OIN team at <oin@okta.com> for assistance.
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md:34]: do we use the word "dimmed" in other places, over "inactive" or "disabled" referring to buttons that exist but cannot be clicked? nbd, as long as its consistent.

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md:34]: Yeah - according to our style guide, which references https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/g/gray-grayed-out, when referring to the appearance of an icon, we should use "dimmed".

File: packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md [hunk 2]
```diff
@@ -19,3 +19,31 @@ Similarly, when you enable a new capability in your application (for example, ad
...
+
+To delete your submission:
+
+1. Click on the delete icon ![Delete submission](/img/icons/delete_can.png "Deletion icon") beside the **Edit** button.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md:38]: no "on" - Just "Click the...."

File: packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md:34]: I am not uber fond of "dimmed". Graham and I are actually currently considering "unavailable" as our official use and straying from Microsoft on this one.  You could then shorten that sentence.....

Comment (IanHakes-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app/update-submission/index.md]: Cleaned up the text to remove dimmed and tidy up the phrasing.

**Pull request merged**